vendor/src/golang.org/x/term/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Go
```

Comment:
Auto-review: the project doesn't seem to use "go.mod", so I blindly cloned the repository:
```
git clone https://github.com/golang/term.git vendor/src/golang.org/x/term
```
... and removed the `.git` directory there.